pkg/bindings/sourcecre.go.tpl

```diff
@@ -320,6 +364,20 @@ func (e *{{.Normalized.Name}}) Error() string {
 
 {{range $event := $contract.Events}}
 
+func (c *{{$contract.Type}}) LogTrigger{{.Normalized.Name}}Log(confidence evm.ConfidenceLevel, values []{{.Normalized.Name}}) (sdk.Trigger[*evm.Log, *evm.Log], error) {
```

Comment:
Wouldn't make more sense to call the second param filters instead of values?

Reply:
fixed